CHANGELOG.md

```diff
@@ -9,8 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- [#286](https://github.com/green-code-initiative/ecoCode/issues/286) [EC83] [C#] Replace Enum ToString() with nameof
 - [#239](https://github.com/green-code-initiative/ecoCode/issues/239) Add new Java rule EC24 : Optimize Database SQL Queries (Clause LIMIT)
+- [#286](https://github.com/green-code-initiative/ecoCode/issues/286) [EC83] [C#] Replace Enum ToString() with nameof
+- [C#27](https://github.com/green-code-initiative/ecoCode-csharp/issues/27) [EC84] [C#] Avoid async void methods
```

Comment:
"C#27" as issue number ? 🤔

Reply:
It's issue 27 but on the csharp repo, not this one, not sure how to display it here

Reply:
"#27" alone would be ok for me. 🤔

Reply:
Ok I changed it. But there's always a risk that another issue with the same id but coming from another eco repo is added, that's why I added C in front of it initially.

Reply:
good idea ! but for me, the most important is the link of issue ... but why not